css/styles.css

```
@@ -0,0 +1,214 @@
+    background-color: var(--very-dark-blue);
+    padding: 1em;
+    border-radius: 10px;
+    display: flex;
+    align-items: end;
+}
+.action_control_button:nth-of-type(2n) {
+    margin: 0 1.5em;
+}
+
+/*
+    Second Box
+*/
+.usage_container {
+    margin: 0;
+    padding: 0;
+}
+
+.usage {
+    border-radius: 10px;
+    padding: 3em;
+    margin: 1.1em;
+    position: relative;
+}
+
+.usage_scale {
+    display: flex;
+    justify-content: space-between;
+}
+
+.usage_pop {
+    background-color: #FFF;
+    color: var(--very-dark-blue);
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 180px;
+    max-width: 180px;
+    min-height: 72px;
+    max-height: 72px;
+}
+
+.usage_pop_text {
+    color: var(--grayish-blue);
+    margin-left: 0.5em;
+    text-transform: capitalize;
+}
+
+.usage_bar {
+    background-color: var(--very-dark-blue);
+    height: 1.4em;
+    border-radius: 20px;
+    margin: 1em 0 0.5em 0;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+}
+
+.usage_bar_consumption {
+    height: 1em;
+    width: 81.5%;
+    border-radius: 20px;
+    background-image: linear-gradient(to right, hsl(6, 100%, 80%), hsl(335, 100%, 65%));
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-left: 0.2em;
+}
+
+.usage_bar_point {
+    background-color: #FFFFFF;
+    border-radius: 10px;
+    height: 0.8em;
+    width: 0.8em;
+    margin-right: 0.1em;
+}
+
+/*
+    mobile only styles
+ */
+@media only screen and (max-width: 940px) {
+    body {
+        background-image: url("../images/bg-mobile.png");
+        background-size: cover;
+    }
+
+    .container {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .action {
+        width: 10em;
+    }
+
+    .usage_container {
+        width: 10em;
+    }
+
+    .usage_pop {
+        position: absolute;
+        bottom: -3.5em;
+        left: 6em;
+    }
+
+}
+
+/*
+    desktop only style
+*/
+@media only screen and (min-width: 941px) {
+    body {
+        background-image: url("../images/bg-desktop.png");
+        background-position: bottom;
+        background-size: 100% 50%;
+    }
+
+    .container {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .container_usage {
+        min-height: 100%;
+        display: flex;
+        align-items: flex-end;
+    }
+
+    .usage {
+        width: 38.5em;
+    }
+
+    .usage_pop {
+        position: absolute;
+        top: -22px;
+        right: -22px;
+    }
+
+    .usage_pop_desktop_pointer {
+        content: "";
+        position: absolute;
+        top: -3.5em;
+        right: 3em;
+
+        min-width: 180px;
+        max-width: 180px;
+        min-height: 50px;
+        max-height: 50px;
+
+        border-color: transparent white transparent transparent;
+        border-width: 22px;
+        border-style: solid;
+        border-radius: 0 20px 0 0;
```

```suggestion
        border-radius: 0 10px 0 0;
```
Je pense que tes bordures sont trop larges